language/move-lang/tests/move_check/parser/byte_string_invalid_hex.exp

```diff
@@ -0,0 +1,8 @@
+error: 
+
+   ┌── tests/move_check/parser/byte_string_invalid_hex.move:3:21 ───
+   │
+ 3 │             b"libra \x{0"
+   │                     ^^^ invalid character in numeric character escape: '\x{0'
```

Comment:
See earlier comment. With that message, I would expect the quoted value to show the invalid character. Also, the loc reported here does not include the "0" even though the "0" is printed in the message. I would fix the loc to include the entire (incomplete) escape sequence and change the message to say that the escape is not complete (or something to that effect)

Reply:
Fixed it.